### ISSUE TITLE
virtualbox-iso Unknown option: --sataportcount with packer 1.3.3 and VirtualBox 6.0.2

### ISSUE THREAD

**Issue by cesarjorgemartinez** [hashicorp/packer#7213]
Hi,

I'm trying to perform an update from Packer 1.3.1 to 1.3.3, using the last VirtualBox stable version 6.0.2, but it is failing with error: "VBoxManage.exe: error: Unknown option: --sataportcount".

I'm not sure, but, is it a Packer error, or Virtualbox error?

Regards,
Cesar Jorge

- Packer version from `packer version`: 1.3.3

- Host platform: Windows 10

- Debug log output from `PACKER_LOG=1 packer build template.json`.

```
==> virtualbox-iso: Retrieving ISO
    virtualbox-iso: Using file in-place: file:///C:/cygwin64/home/admin/automatevmimages/isos/CentOS-7-x86_64-Minimal-1810.iso
==> virtualbox-iso: Starting HTTP server on port 10084
==> virtualbox-iso: Creating virtual machine...
==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Error creating disk controller: VBoxManage error: Oracle VM VirtualBox Command Line Management Interface Version 6.0.2
==> virtualbox-iso: (C) 2005-2019 Oracle Corporation
==> virtualbox-iso: All rights reserved.
==> virtualbox-iso:
==> virtualbox-iso: Usage:
==> virtualbox-iso:
==> virtualbox-iso: VBoxManage storagectl       <uuid|vmname>
==> virtualbox-iso:                             --name <name>
==> virtualbox-iso:                             [--add ide|sata|scsi|floppy|sas|usb|pcie]
==> virtualbox-iso:                             [--controller LSILogic|LSILogicSAS|BusLogic|
==> virtualbox-iso:                                           IntelAHCI|PIIX3|PIIX4|ICH6|I82078|
==> virtualbox-iso:                             [             USB|NVMe]
==> virtualbox-iso:                             [--portcount <1-n>]
==> virtualbox-iso:                             [--hostiocache on|off]
==> virtualbox-iso:                             [--bootable on|off]
==> virtualbox-iso:                             [--rename <name>]
==> virtualbox-iso:                             [--remove]
==> virtualbox-iso:
==> virtualbox-iso: VBoxManage.exe: error: Unknown option: --sataportcount
```

**Comment by rickard-von-essen** [comment]
Duplicates #7087, which is already fixed on master and scheduled for 1.3.4.

In the referenced issue there are patched builds of Packer that you can use.

**Comment by rickard-von-essen** [comment]
[patched builds of Packer](https://github.com/hashicorp/packer/issues/7087#issuecomment-446012457)
